### PR TITLE
Invite connection options

### PR DESCRIPTION
**Pull request by yipingliao**
Fix infinite scroll problem:
https://app.asana.com/0/15014070792041/15074070184291

Add link to users with 0 friends and 0 connections to invite friends:
https://app.asana.com/0/15014070792041/15038735966102

@andrewconner @stkem @2is10

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/gruntfile.js [hunk 1]
```diff
@@ -59,13 +59,13 @@ module.exports = function (grunt) {
           'lib/jquery.mousewheel/jquery.mousewheel.js',
           'lib/antiscroll/antiscroll.js',
           //'lib/angular-antiscroll/angular-antiscroll.js',
```
Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/gruntfile.js]: I noticed a couple of these commented out files. Anyone object to removing them if we check that they are no longer  needed (will check bower.json, etc.)

Comment (andrewconner) [inline kifi-backend/modules/shoebox/angular/gruntfile.js]: Yep, remove it!

**Comment by andrewconner** [comment]
Awesome!

File: kifi-backend/modules/shoebox/angular/managed-lib/angular-smart-scroll/.bower.json [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  "_release": "ea4f507976",
+  "_resolution": {
+    "type": "branch",
+    "branch": "master",
+    "commit": "ea4f507976f634473ea915ccd0e0575b0d036df9"
+  },
+  "_source": "git://github.com/joonho1101/angular-smart-scroll.git",
+  "_target": "*",
+  "_originalSource": "angular-smart-scroll"
+}
```
Comment (2is10) [inline kifi-backend/modules/shoebox/angular/managed-lib/angular-smart-scroll/.bower.json]: we end text files with a newline. thanks.

**Comment by 2is10** [comment]
:+1: great work!

File: kifi-backend/modules/shoebox/angular/managed-lib/angular-smart-scroll/.bower.json (hunk 2 above)
Comment (stkem) [inline kifi-backend/modules/shoebox/angular/managed-lib/angular-smart-scroll/.bower.json]: It's not just us, there are some good reasons to: http://stackoverflow.com/questions/729692/why-should-files-end-with-a-newline

**Pull request merged**